ccd-definition/ComplexTypes/2_direction.json

```diff
@@ -62,6 +62,28 @@
     "SecurityClassification": "Public",
     "FieldShowCondition": "directionNeeded!=\"No\""
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Direction",
+    "ListElementCode": "parentsAndRespondentsAssignee",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "ParentsAndRespondentsDirectionAssignee",
+    "ElementLabel": "Assignee",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "directionNeeded!=\"No\"",
+    "Comment": "This field is used for capturing parents and respondents assignee on CMO"
```

Comment:
Comment? Is that kept in XLS file or just transient value in JSON for own own benefits?

Reply:
It is a transient value, just added to explain the field or whatever it is attached to